domains/sdp.json

```diff
@@ -0,0 +1,13 @@
+{
+    "owner": {
+        "username": "Stef-00012",
+        "email": "admin@stefdp.is-a.dev",
+        "discord": "694986201739952229"
+    },
+    "record": {
+        "A": [
+            "46.248.190.217"
+        ],
+        "TXT": "cuttly-verification-site-77f386c161a3f598440c290aae943430af0ecb97-b2a45cef44"
```

Comment:
We don't support TXT verification.

Reply:
removed the txt verification and changed the A record to a host i use already on other domains (so shouldn't be an issue)